### ISSUE TITLE
Request for v11

### ISSUE THREAD

**Issue by davelacorneille** [Kekilla0/World-Scripter#10]
Do you think you will update this module to V11?

Thank you!

**Comment by trioderegion** [comment]
Hi hi, life has a habit of getting in the way. To my knowledge, there are no specific plans for discontinuation.

Currently, we are relying a bit on community contributions as both the owner (@Kekilla0 ) and the maintainer (myself) have very limited time on our hands for hobby development.

World Scripter is still loved and not forgotten, I can offer that 💖.  I am going to leave this issue open for others as well.

**Comment by moxSedai** [comment]
What needs to change for the module to properly work in V11?

**Comment by moxSedai** [comment]
Will this ever be updated, or is it dead?

**Comment by Kekilla0** [comment]
I will get around to it soon. I didn't know if anyone still used it.

**Comment by dsakura** [comment]
I'm trying to use it. Great module o/

**Comment by Rakarck** [comment]
Yes, there are still people who use this module, it is the only one that has this functionality.

Waiting for version 11 and 12

congratulations!

**Comment by SilvesTheRog** [comment]
Would really love to see a V12 version one day!

**Comment by trioderegion** [comment]
With modern foundry, world scripter is a relic of the past and likely will not receive any updates.
